homeassistant/components/vesync/manifest.json

```diff
@@ -8,7 +8,7 @@
     "@thegardenmonkey"
   ],
   "requirements": [
-    "pyvesync==1.1.0"
+    "pyvesync-api==1.2.0"
```

Comment:
We don't change library unless there's a clear reason or benefit with the new library. If the existing library maintainer wants to continue support us there should be significant reasons to change library.

Please first discuss this with the existing library maintainers.

Reply:
Thanks @MartinHjelmare I was able to get the existing library maintainers to bump the version in their library. Updated this PR to reflect existing library, just the version is bumped to fix the issue.